src/bonsai/bonsai/bim/module/sequence/__init__.py

```diff
@@ -81,6 +81,7 @@
     operator.GenerateGanttChart,
     operator.GuessDateRange,
     operator.GoToTask,
+    operator.ImportAlignmentCSV,
```

Comment:
This should be in module/geometry perhaps, or create your own new module like module/alignment (probably better long term as it grows in functionality).

Reply:
@Moult I removed all the Bonsai changes I made under module/sequence and created module/alignment.
In module/alignment I have __init__.py and operator.py because these were the only files I changed in module/sequence.

The menu command for Alignment no longer shows up in the Import menu.

What am I missing?

The files are here:
https://github.com/IfcOpenShell/IfcOpenShell/tree/RAB_alignment_api/src/bonsai/bonsai/bim/module/alignment

Reply:
This is now working from its own module